auctions/english-auction/english-auction.config.ts

```diff
@@ -13,6 +15,42 @@ function getEnvName() {
   return '.env';
 }
 
+/**
+ * Use event output data to replace
+ * on the provide path env the new
+ * contract ids.
+ *
+ * It uses the name inform on the config.contracts.name
+ * as a key to the new value. If it didn't found the key
+ * on the provide path nothing happens
+ */
+export async function replaceEventOnEnv(path: string, event: Event) {
+  if (event.type === Commands.deploy || event.type === Commands.run) {
+    log(`Reading file from ${path}`);
```

Comment:
```suggestion
```

Remove logs in prod?

Reply:
hmmm I feel like some of the logs in the application are useful.  For example, informing the dev which nfts are minted to which wallet addresses.  But I guess we could provide that info in documentation.